src/os/bluestore/BlueStore.cc

```diff
@@ -6909,8 +6909,11 @@ int BlueStore::_open_super_meta()
       ::decode(bluefs_extents, p);
     }
     catch (buffer::error& e) {
-      derr << __func__ << " unable to read bluefs_extents" << dendl;
-      return -EIO;
+      string kv_backend = cct->_conf->bluestore_kvbackend;
```

Comment:
oic, so i think a better fix would be to check `cct->_conf->bluestore_bluefs` before trying to read `bluefs_extents` from db. in other words, to guard the "bluefs alloc" block with `if (cct->_conf->bluestore_bluefs)`.